NitroxModel/Extensions.cs

```diff
@@ -85,6 +86,30 @@ public static string AsByteUnitText(this uint byteSize)
         return num + suf[place];
     }
 
+    /// <summary>
+    ///     Calls an action if an error happens. If null, logs the error as-is.
+    /// </summary>
+    /// <remarks>
+    ///     Use this for fire-and-forget tasks so that errors aren't hidden when they happen.
+    /// </remarks>
+    public static Task ContinueWithHandleError(this Task task, Action<Exception> onError = null) =>
+        task.ContinueWith(t =>
+        {
+            if (t is not { IsFaulted: true, Exception: { } ex })
+            {
+                return;
+            }
+
+            if (onError != null)
+            {
+                onError(ex);
```

Comment:
I think this design might lead to someone forgetting to Log the error at some point inside the callback

Reply:
Can we maybe have Log.Error be the default callback ?

Reply:
I don't see an issue here. If you don't pass onError it will already default to Log.Error. If you pass an onError handler but forget to log, well, don't. Or do, if you want to handle the error in a different way.